src/action-creators/subCategorizeOne.js

```diff
@@ -0,0 +1,27 @@
+import { store } from '../store.js'
+
+// action-creators
+import { newThought } from './newThought'
+
+// constants
+import {
+  RENDER_DELAY,
+} from '../constants.js'
+
+// util
+import {
+  contextOf,
+  head,
+} from '../util.js'
+
+export const subCategorizeOne = () => (dispatch) => {
+  const { cursor } = store.getState()
+  const { rank } = dispatch(newThought({ insertBefore: true }))
+  setTimeout(() => {
+    dispatch({
+      type: 'existingThoughtMove',
+      oldPath: cursor,
+      newPath: contextOf(cursor).concat({ value: '', rank }, head(cursor))
+    })
+  }, RENDER_DELAY)
```

Comment:
I'm guessing the `setTimeout` was necessary, but I wonder if this would work by dispatching another thunk instead of the `setTimeout`? That would be a cleaner option.

Reply:
I understand the u

> I think this is ready to merge. What do you think?
> 
> We can refactor the `setTimeout` calls in a separate issue if we want.

Sure. I have pushed some changes. It's ready to merge. And yes we should have a separate issue for `setTimeout` calls problem.